app/models/switch.rb

```diff
@@ -1,3 +1,7 @@
+# this is a HostVirtualSwitch
+# as opposed to: ManageIQ::Providers::InfraManager::DistributedVirtualSwitch
+# Switch uses host to link to an Ems
+# DistributedVirtualSwitch directly links to an ems
```

Comment:
I think this makes it more confusing because it makes it sound like Switch literally is a HostVirtualSwitch, and that DistributedVirtualSwitch is not a Switch.  In fact, both of those are subclasses of Switch.